docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
+
+We support linking to the source code on GitHub, or any other VCS provider.
+
+![Link to GitHub](../../images/guide/link-to-github.gif)
+
+## Usage
+
+To enable source code linking with Logfire, you need to configure your project. Here's how:
+
+```python
+import logfire
+
+logfire.configure(
+    code_source=logfire.CodeSource(
+        repository='https://github.com/pydantic/logfire',
+        revision='main',
+        root_path='.',
+    )
+)
+```
+
+You can check more about it on the [`logfire.CodeSource`][logfire.CodeSource] on our API reference.
+
+| Parameter | Description |
+| --- | --- |
+| `repository` | The URL of the repository e.g. `https://github.com/pydantic/logfire` |
+| `revision` | The specific branch, tag, or commit hash to link to e.g. `main` |
+| `root_path` | The path to the root of the repository. If your code is in a subdirectory, you can specify it here. |
+
+## Alternative Configuration
```

Comment:
Seems like this pattern comes up a lot (re our settings vs otel env vars). Maybe we should start using tab panels to explain how to set things (one tab for logfire, one tab for otel env var)?

Reply:
Yes, that would be great.